yew/src/agent/worker/queue.rs

```diff
@@ -0,0 +1,48 @@
+use std::any::TypeId;
+use std::cell::{RefCell, RefMut};
+use std::collections::{hash_map, HashMap, HashSet};
+
+/// Thread-local instance used to queue worker messages
+pub struct Queue {
+    loaded_agents: RefCell<HashSet<TypeId>>,
+    msgs_queue: RefCell<HashMap<TypeId, Vec<Vec<u8>>>>,
```

Comment:
`msg_queue` sounds better to me, but this is just personal preference
```suggestion
    msg_queue: RefCell<HashMap<TypeId, Vec<Vec<u8>>>>,
```